omod/src/main/webapp/resources/htmlforms/003-HIVCare-ARTCard-ClinicalAssessmentForm.xml

```diff
@@ -0,0 +1,1422 @@
+                                    
+                                </excludeIf>
+                            </div>
+                            <div class="row">
+                                <excludeIf velocityTest="$patient.age &lt; 9">
+                                    <excludeIf velocityTest="$patient.gender =='M'">
+                                       <div class="col-md-4">
+                                            <span class="ARTYesNoEnableDisableChild3">
+                                                <label>Age in months</label>
+                                                <obs conceptId="99270" />
+                                            </span>
+                                        </div>
+                                        
+                                        <div class="col-md-4">
+                                        <span class="ARTYesNoEnableDisableChild2">
+                                           
+                                            <label>ANC #</label>
+                                            <patient field="identifier" identifierTypeId="b0cc255e-ef57-4266-839e-0df7f5099eb9"
+                                                     required="false"/>
+                                        </span>
+                                        </div>
+                                    </excludeIf>
+                                </excludeIf>
+                            </div>
+                                    
+
+                            <excludeIf velocityTest="$patient.age &lt; 9">
+                               
+                                    <!-- <div class="row">
+                                        <div class="col-md-3">
+                                        <span id="checkboxRadio">
+                                            <span class="ARTYesNoEnableDisable1">
+                                                <label>Family Planning? </label>
+                                                <obs conceptId="90238" answerConceptIds="90082,90081" answerLabels="Yes,No" />
+                                            </span>
+                                        </span>
+                                        </div>
+                                    </div> -->
+                                 <br/>
+                                <div class="row">
+                                    <div class="col-md-6">
+                                        <label>Is on family Planning?</label>
+                                    <div class="form-check form-check-inline">
+                                      <input class="form-check-input" type="radio" name="inlineRadioOptions" id="inlineRadio1" value="option1" />
+                                      <label class="form-check-label" for="inlineRadio1">No</label>
+                                    </div>
+                                    <div class="form-check form-check-inline">
```

Comment:
Why use input What happened to the <obs> tag

Reply:
They are place holders for now, this is still WIP